### DISCUSSION TITLE
Extreme Poisson Ration Simulation

### DISCUSSION THREAD

**Discussion by hyang14** [sofa-framework/sofa#3094] [Write / Run a simulation] [answered]
Hi all. I was trying to write a simulation for a cubic human tissue as a demo.  After a quick look at some research papers, one suggested that human tissue should be treated as incompressible which has a Poisson Ration of near 0.5. Without prior knowledge, I used 0.5 for my FF. Later I find out, that the whole model disappears once I click animate unless I fix every point in the model. 

Just curious, why would the whole model disappear when I set the Poisson ratio to 0.5? 

Thanks very much!

**Comment by alxbilger** [comment, marked as answer]
@HiroakiYo 
0.5 is a theoretical value that cannot be reached numerically. The reason is that the material stiffness depends on

$\frac{1}{1-2*poissonRatio}$

You can observe that setting the Poisson's ratio to 0.5 leads to a division by zero.
In the next version of SOFA, you will be warned that 0.5 is not valid value.

Alex

**Comment by hyang14** [comment]
Thanks so much for the explanation, Alex!
